gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformerExec.scala

```diff
@@ -142,9 +160,9 @@ case class WholeStageTransformerExec(child: SparkPlan)(val transformStageId: Int
       printNodeId,
       indent)
     if (verbose && planJson.nonEmpty) {
-      append(prefix + "Substrait plan:\n")
-      append(planJson)
-      append("\n")
+//      append(prefix + "Substrait plan:\n")
```

Comment:
why comment?

Reply:
it's for debugging. will remove before merging